### PR TITLE
fix: fix master CI FAIL

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: fix master CI FAIL |
| 🇨🇳 Chinese |          - |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at f25a1d3</samp>

Updated `dayjs` dependency to fix date issues. This change affects the `package.json` file.

### <samp>🤖 Generated by Copilot at f25a1d3</samp>

* Update `dayjs` dependency to fix date issues ([link](https://github.com/ant-design/ant-design/pull/44955/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L124-R124))

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
     "@rc-component/trigger": "^1.15.6",
     "classnames": "^2.2.6",
     "copy-to-clipboard": "^3.2.0",
-    "dayjs": "^1.11.1",
+    "dayjs": "^1.11.10",
```
[the remark this answers is not in the document]
Comment (afc163) [reply package.json:124]: 我意思不改这个，让 dayjs 的 semver 兼容范围更广一点。

业务代码里如果写了 `"dayjs": "1.11.6"` 依赖的话，会导致引入两个版本的 dayjs。可以避免这种情况。

[the remark this answers is not in the document]
Comment (li-jia-nan) [reply package.json:124]: 现在 semver 已经是用 `^` 做前缀了，这里其实改不改都一样，如果要更宽的话，就要改成 `*` 做版本前缀了

[the remark this answers is not in the document]
Comment (afc163) [reply package.json:124]: 不一样的，`^1.11.1` 比 `^1.11.10` 要宽一些，没有卡点问题这里可以不缩的。

[the remark this answers is not in the document]
Comment (li-jia-nan) [reply package.json:124]: 懂了，那这个可以改回去，另外我看到快照的变化在另一个 pr 里已经修了，这个 pr 可以 close 了

**Pull request closed without merging**